src/modules/dashboard/components/ActionsPage/InputStorageWidget/InputStorageWidget.tsx

```diff
@@ -82,88 +82,79 @@ const InputStorageWidget = ({
     }
   }, [storageSlot, colonyAddress, fetchStorageSlotValue]);
 
+  /*
+   * Prevent entering a new line
+   *
+   * Based on specs we need to **fake** a multi-line input, but one
+   * you can't actually create a new line manully, just if the text
+   * overflows...
+   *
+   * While we're at it we also disable the space bar, since no value
+   * that can be entered requires a space (it's just a litte nice-to-have)
+   */
+  const handleOnKeyDown = useCallback((event) => {
+    if (event.key === ENTER || event.code === SPACE) {
+      event.preventDefault();
+    }
```

Comment:
Note that this change will likely cause a merge conflict for @chinins who started working on #2713

Reply:
Well, I guess I'll be rebasing the coin-machine-redux on master anyway :)